attrs_strict/_error.py

```diff
@@ -89,3 +90,17 @@ def __str__(self):
         )
 
         return self._render(error)
+
+
+def _is_newtype(type_):
+    """Checks if `type` is :func:`typing.NewType`."""
+    return (hasattr(type_, "__qualname__") and
+            type_.__qualname__ == "NewType.<locals>.new_type")
+
+
+def _format_type(type_):
+    """Returns a string to pretty display a given `type_` to the user."""
+    if _is_newtype(type_):
+        return "NewType({}, {})".format(type_.__name__, type_.__supertype__)
+    else:
```

Comment:
You don't need the else here.

Reply:
correct, I prefer it with else to indicate it is not an error case but a decition, anyway I removed it